subsys/bluetooth/mesh/cdb.c

```diff
@@ -1030,6 +1030,14 @@ struct bt_mesh_cdb_app_key *bt_mesh_cdb_app_key_alloc(uint16_t net_idx,
 	struct bt_mesh_cdb_app_key *key;
 	int i;
 
+	for (i = 0; i < ARRAY_SIZE(bt_mesh_cdb.app_keys); ++i) {
```

Comment:
I'm pretty sure it is possible to do in the single loop.

Reply:
Im affraid it will be ugly and less readable. But I can give it a go. Unless you have a specific approach in mind?

Reply:
I changed it.